cli/src/main/resources/help/server.adoc

```diff
@@ -0,0 +1,67 @@
+SERVER(1)
+=========
+:doctype: manpage
+
+
+NAME
+----
+server - server configuration and state management.
+
+
+DESCRIPTION
+-----------
+
+
+
+SYNOPSIS
+--------
+*server report*
+
+*server connector ls*
+
+*server connector describe* 'connector-name'
+
+*server connector start* 'connector-name'
+
+*server connector stop* 'connector-name'
+
+*server connector ipfilter ls* 'connector-name'
+
+*server connector ipfilter set* 'connector-name' --rules='[ACCEPT|REJECT]/cidr',...
+
+*server connector ipfilter clear* 'connector-name'
+
+
+SERVER CONNECTOR IPFILTER OPTIONS
+---------------------------------
+
+*--rules*='[ACCEPT|REJECT]/cidr',...::
+One or more IP filtering rules.
+
+
+EXAMPLES
+--------
+
+`server report` +
+Obtains a server report, including information about network, threads, memory, etc.
+
+`server connector ls` +
+Lists all available connectors on the server.
+
+`server connector describe endpoint-default` +
+Shows information about the specified connector, including host, port, local and global connections, IP filtering rules.
+
+`server connector start endpoint-default` +
+Starts a connector so that it can accept connections across the cluster. This operation can only be performed on connectors which are directly bound to a socket.
+
+`server connector stop endpoint-default` +
+Stops a connector dropping all established connections across the cluster. This operation can only be performed on connectors which are directly bound to a socket.
+
+`server connector ipfilter ls endpoint-default` +
+Lists all IP filtering rules active on a connector across the cluster.
+
+`server connector ipfilter set endpoint-default --rules=ACCEPT/192.168.0.0/16,REJECT/10.0.0.0/8`
```

Comment:
wouldn't be a better user experience to create an "accept" and "reject" command instead? Overwriting all the rules seems a little bit harsh 

ps.1 Or make it more like the iptables:
`server connector ipfilter add my-connect accept|reject 10.0.0.0/8`
`server connector ipfilter delete my-connect accept|reject 10.0.0.0/8`
or (probably too much)
`server connector ipfilter my-connect -A|-D  -j accept|reject -s 10.0.0.0/8`

ps.2 what about moving ipfilter to server's subcommand?
`server ipfilter my-connector ...`

Reply:
> wouldn't be a better user experience to create an "accept" and "reject" command instead?

Because order matters and I don't want to have to deal with manipulating before/after stuff.

> ps.1 Or make it more like the iptables:

Again, it would need manipulating order.

> ps.2 what about moving ipfilter to server's subcommand?

Because ipfilters are a subfeature of connectors

Reply:
fair enough :)